examples/simple_system/README.md

```diff
@@ -113,6 +113,21 @@ To run the simulator:
 
 Pass `-gui` to use the DVE GUI.
 
+## Simulating with Riviera-PRO
+
+To build and run Simple System run:
+
+```
+fusesoc --cores-root=. run --target=sim --tool=rivierapro lowrisc:ibex:ibex_simple_system --RV32M=1 --RV32E=0 \
+  --SRAM_INIT_FILE=<sw_vmem_file> --vsim_options=+ibex_tracer_file_base=test_results
```

Comment:
ibex_tracer_file_base should actually be a plusarg known to fusesoc. That's currently not the case and something we should fix. In the meantime (and for consistency with other examples in this file), can you remove the plusarg?